src/components/v5/common/WidgetCards/styles.ts

```diff
@@ -0,0 +1,6 @@
+import { tw } from '~utils/css/index.ts';
+
+export const wrapperClassName = tw`flex min-h-[7.4rem] min-w-[16.5rem] items-center justify-between gap-2 rounded-lg border px-6 py-6`;
```

Comment:
do all of these need to be in a separate file? we can have them inside the component className, unless we want something to do with them later on